### PR TITLE
DDF-3386 Upgrades yarn and frontend-maven-plugin

### PR DESCRIPTION
**Pull request by andrewkfiedler**
See https://github.com/codice/ddf/pull/2513/

@oconnormi 
@clockard 
@vinamartin

### CODE REVIEW

**Comment by oconnormi** [comment]
build now

**Comment by LinkMJB** [comment]
build now

**Comment by LinkMJB** [comment]
Strange that it still retrieved Yarn from Github, despite being advised to use the local proxy via: `-Dyarn.download.root="http://nexus3.phx.connexta.com/repository/yarnpgk/"`

> 15:25:58 [INFO] Installed node locally.
> 15:25:58 [INFO] Installing Yarn version v1.2.1
> 15:25:58 [INFO] Downloading https://github.com/yarnpkg/yarn/releases/download/v1.2.1/yarn-v1.2.1.tar.gz to /root/.m2/repository/com/github/eirslett/yarn/1.2.1/yarn-1.2.1.tar.gz
> 15:25:58 [INFO] No proxies configured
> 15:25:58 [INFO] No proxy was configured, downloading directly
> 15:25:58 [INFO] Starting audit...

File: pom.xml [hunk 1]
```diff
@@ -352,7 +352,7 @@
                 <plugin>
                     <groupId>com.github.eirslett</groupId>
                     <artifactId>frontend-maven-plugin</artifactId>
-                    <version>1.4.CODICE_1</version>
+                    <version>1.6.CODICE</version>
                     <executions>
                         <execution>
                             <id>install node and yarn</id>
```
Comment (LinkMJB) [inline pom.xml]: The configuration should be updated to allow over-ride of Yarn retrieval location. 

See DDF for reference: 
https://github.com/codice/ddf/blob/master/pom.xml#L514
VERSUS
https://github.com/codice/alliance/blob/cdaecef3ef0458f646a534785c3b4c899b8e75e8/pom.xml#L362

**Comment by bdeining** [comment]
build now

File: pom.xml (hunk 1 above)
Comment (bdeining) [reply pom.xml]: done

**Comment by bdeining** [comment]
build now

File: pom.xml (hunk 1 above)
Comment (LinkMJB) [reply pom.xml]: Thanks @bdeining!

**Pull request merged**